Exiled.API/Features/Player.cs

```diff
@@ -2497,16 +2482,8 @@ public void ResetInventory(IEnumerable<ItemType> newItems)
         {
             ClearInventory();
 
-            Timing.CallDelayed(
```

Comment:
Player items are sent the frame after they are removed, therefore you need to wait at least 1 frame between removing the last one and adding a new one, or the client could hit a race condition where it's trying to add an item into a slot index that's already occupied.

Reply:
same "i will like to request someone to test it at least to be 100% sure"

because they have change how Item was sync so i am pretty curious